<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogColorSet</class>
 <widget class="QDialog" name="CDialogColorSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像调节</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-size:15px;
	font-family:Microsoft Yahei;	
}
QDialog{background-color: rgb(100, 100, 100);
border-width:2px;
border-color:rgb(255,0,0);
}
QWidget{color:rgb(255, 255, 255)}
</string>
  </property>
  <widget class="QFrame" name="frameSysBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border-image:url(:/Skin/bg-above2.png);
}
QLabel
{
border-image:null;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="labelTitle">
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>170</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelTitle
{
	color: rgb(255, 255, 255);
	font-size:18px;
	font-family:Microsoft Yahei;	
}</string>
        </property>
        <property name="text">
         <string>图像调节</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="logo">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Skin/logo.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QFrame" name="frame_sysmenu">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{border-image:null;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="BtnClose">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>关闭</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/close.png)0 0 repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/close.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/close.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/close.png)90 0 0 0 repeat
}
</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnDefault">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
   </property>
   <property name="text">
    <string>恢复默认</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="horizontalSpacing">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>亮度:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="CMySlider" name="SliderBright">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	border-image:url(:/Skin/bright.png);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}
/*
QSlider::sub-page:horizontal{ 	

border-image:url(:/Skin/bright.png);
	margin: 2px 0px 2px 0px;
}*/</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>饱和度:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="CMySlider" name="SliderSaturation">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	border-image:url(:/Skin/saturation.png);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}
/*
QSlider::sub-page:horizontal{ 	

border-image:url(:/Skin/bright.png);
	margin: 2px 0px 2px 0px;
}*/</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>对比度:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="CMySlider" name="SliderContrast">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	border-image:url(:/Skin/constrast.png);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}
/*
QSlider::sub-page:horizontal{ 	

border-image:url(:/Skin/bright.png);
	margin: 2px 0px 2px 0px;
}*/</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>色度:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="CMySlider" name="SliderHue">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	border-image:url(:/Skin/hue.png);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}
/*
QSlider::sub-page:horizontal{ 	

border-image:url(:/Skin/bright.png);
	margin: 2px 0px 2px 0px;
}*/</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBoxSharp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>启用锐化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="CMySlider" name="SliderSharp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>354</y>
     <width>271</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>10</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	/*border-image:url(:/Skin/bright.png);*/
	background-color:rgb(200, 200, 200);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}
/*
QSlider::sub-page:horizontal{ 	

border-image:url(:/Skin/bright.png);
	margin: 2px 0px 2px 0px;
}*/</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="CMySlider" name="SliderDecNoise">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>271</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>10</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
  /*  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #A1A1A1, stop:1 #B4B4B4);
	background-color: rgb(200, 200, 200);*/
	/*border-image:url(:/Skin/bright.png);*/
	background-color:rgb(200, 200, 200);
    margin: 2px 0 2px;
}

QSlider::handle:horizontal {
	border-image: url(:/Skin/thum.png);

    width: 16px;
 	height: 12px;
    margin: -3px 0px -7px 0px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}
/*
QSlider::sub-page:horizontal{ 	

border-image:url(:/Skin/bright.png);
	margin: 2px 0px 2px 0px;
}*/</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDecNoise">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>396</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>启用降噪</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMySlider</class>
   <extends>QSlider</extends>
   <header>cmyslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sisecplayer.qrc"/>
 </resources>
 <connections/>
</ui>
